modules/apps/rate-limiting/keeper/flow.go

```diff
@@ -17,7 +17,7 @@ func (k Keeper) GetChannelValue(ctx sdk.Context, denom string) sdkmath.Int {
 }
 
 // Adds an amount to the flow in either the SEND or RECV direction
-func (k Keeper) UpdateFlow(rateLimit types.RateLimit, direction types.PacketDirection, amount sdkmath.Int) error {
+func UpdateFlow(rateLimit types.RateLimit, direction types.PacketDirection, amount sdkmath.Int) error {
```

Comment:
I wonder why this is not a function on RateLimit directly? Would make more sense than passing in rateLimit and modifying it.